### ISSUE TITLE
Need help with SceneJs rendering

### ISSUE THREAD

**Issue by pvhuybao2210** [opensourceBIM/BIMserver#1061]
Hello, everyone.
I'm wondering if I can add some circles to canvas like this. 

![image](https://user-images.githubusercontent.com/32448370/69218251-ea326480-0ba2-11ea-8e43-c1e799f2b7c1.png).

Currently I'm using div tags to display labels like Scene.Js example, but I would like to add them as a part of the canvas.
I don't know where to start. Do you have any idea?

Thank in advance!

**Comment by rubendel** [comment]
I don't think this is a relevant issue for BIMserver. Moving to BIMsurfer repository.
